packages/api/src/models/sensorModel.js

```diff
@@ -125,7 +125,7 @@ class Sensor extends Model {
   static async getSensorLocationBySensorIds(locationIds = []) {
```

Comment:
@jagDhillon077 can you refactor this function name as well? Maybe `getSensorLocationById` or `getSensorLocationByLocationId` ?